spec/models/manageiq/providers/oracle_cloud/cloud_manager/refresher_spec.rb

```diff
@@ -0,0 +1,102 @@
+describe ManageIQ::Providers::OracleCloud::CloudManager::Refresher do
+  let!(:ems) do
+    FactoryBot.create(:ems_oracle_cloud_with_vcr_authentication)
+  end
+
+  describe "#refresh" do
+    context "full refresh" do
+      it "Performs a full refresh" do
+        1.times do
```

Comment:
1 time?

Reply:
Ah damn, forgot to put back after recording vcrs

Reply:
There we go, fixed